sdk/nodejs/src/request.ts

```diff
@@ -47,6 +54,56 @@ export async function getEnvironmentConfig(
     })
 }
 
+export async function getAllFeatures(user: DVCPopulatedUser, envKey: string): Promise<AxiosResponse> {
+    return await post({
+        url: `${BUCKETING_URL}${HOST}${FEATURES_PATH}`,
+        headers: { 'Authorization': envKey, 'Content-Type': 'application/json' },
```

Comment:
seems like there's a bit more abstraction you could get out of sending requests here.

all of these need the same headers and base url

Reply:
simplified this a bit